src/main/java/org/jabref/gui/preferences/ExternalTab.java

```diff
@@ -187,6 +224,9 @@ public void storeSettings() {
         prefs.putBoolean(JabRefPreferences.USE_DEFAULT_CONSOLE_APPLICATION, defaultConsole.isSelected());
         prefs.put(JabRefPreferences.CONSOLE_COMMAND, consoleCommand.getText());
         prefs.put(JabRefPreferences.ADOBE_ACROBAT_COMMAND, adobeAcrobatReaderPath.getText());
+        prefs.putBoolean(JabRefPreferences.USE_DEFAULT_FILE_BROWSER_APPLICATION, defaultFileBrowser.isSelected());
+        prefs.put(JabRefPreferences.FILE_BROWSER_COMMAND, fileBrowserCommand.getText());
```

Comment:
Please add a check that the `fileBrowserCommand` is not empty (otherwise keep using the default file browser). Otherwise there will be users which set the command to be empty and then complain that the "Open folder" feature is broken (user can be really creative with such things ;-) ).